lib/export/coldp/files/name.rb

```diff
@@ -17,8 +17,12 @@ def self.code_field(taxon_name)
     end
   end
 
-  def self.remarks_field(taxon_name)
-    Utilities::Strings.nil_squish_strip(taxon_name.notes.collect{|n| n.text}.join('; ')) # remarks - !! check for tabs
+  def self.remarks(name, name_remarks_vocab_id)
+    if !name.data_attributes.find_by(controlled_vocabulary_term_id: name_remarks_vocab_id).nil?
```

Comment:
See below for setting variable inline with `if`.

Reply:
I see what you mean but I think the line might be too long with it inline. I'll try to use the inline style when possible.